annlite/index.py

```diff
@@ -71,11 +75,17 @@ def __init__(
     ):
         setup_logging(verbose)
 
+        if 'dim' in kwargs:
+            logger.warning(
```

Comment:
the deprecation warning should not be done by the logger but by `warnings`

Reply:
done!